docs/troubleshooting.md

```diff
@@ -194,6 +194,14 @@ Collector as it runs. This is an advanced use-case that should not be needed in
 
 ## Common Issues
 
+To see logs for the Collector:
+
+On a Linux systemd system, logs can be found using `journalctl`:  
+`journalctl | grep otel`
```

Comment:
why here otel then 2 lines bellow otelcol?

Reply:
probably due to multitasking working with customer while at same time making suggestions- feel free to fix as you like...

Reply:
feel free to pick whichever you'd prefer...

Reply:
If it is started as a systemd unit, you can use the `-u` flag to `journalctl` instead of grepping for otel.

Reply:
@jpkrohling what is the full journalctl command that you recommend so I can test it with users?

Reply:
Here's an example when using the otel-collector installed via the RPM package, with the expected output:

```console
$ journalctl -u otel-collector
-- Logs begin at Wed 2021-04-07 14:00:18 CEST, end at Wed 2021-04-28 14:52:07 CEST. --
Apr 28 14:51:41 guarana systemd[1]: Started OpenTelemetry Collector.
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.754+0200        info        service/application.go:281        Starting OpenTelemetry Collector...        {"Version": "v0.25.0", "GitHash": "d93c9d29", "NumCPU": 12}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.754+0200        info        service/application.go:189        Setting up own telemetry...
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.766+0200        info        service/telemetry.go:98        Serving Prometheus metrics        {"address": ":8888", "level": 0, "service.instance.id": "815b7908-712c-427c-89d5-dd>
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.766+0200        info        service/application.go:224        Loading configuration...
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        service/application.go:240        Applying configuration...
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/exporters_builder.go:274        Exporter was built.        {"kind": "exporter", "exporter": "logging"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/pipelines_builder.go:204        Pipeline was built.        {"pipeline_name": "metrics", "pipeline_datatype": "metrics"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/pipelines_builder.go:204        Pipeline was built.        {"pipeline_name": "traces", "pipeline_datatype": "traces"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:230        Receiver was built.        {"kind": "receiver", "name": "jaeger", "datatype": "traces"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:230        Receiver was built.        {"kind": "receiver", "name": "zipkin", "datatype": "traces"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:230        Receiver was built.        {"kind": "receiver", "name": "prometheus", "datatype": "metrics"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:230        Receiver was built.        {"kind": "receiver", "name": "otlp", "datatype": "traces"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:230        Receiver was built.        {"kind": "receiver", "name": "otlp", "datatype": "metrics"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:230        Receiver was built.        {"kind": "receiver", "name": "opencensus", "datatype": "traces"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:230        Receiver was built.        {"kind": "receiver", "name": "opencensus", "datatype": "metrics"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        service/service.go:155        Starting extensions...
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/extensions_builder.go:53        Extension is starting...        {"kind": "extension", "name": "health_check"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        healthcheckextension/healthcheckextension.go:41        Starting health_check extension        {"kind": "extension", "name": "health_check", "config">
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/extensions_builder.go:59        Extension started.        {"kind": "extension", "name": "health_check"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/extensions_builder.go:53        Extension is starting...        {"kind": "extension", "name": "pprof"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        pprofextension/pprofextension.go:79        Starting net/http/pprof server        {"kind": "extension", "name": "pprof", "config": {"TypeVal":"pprof">
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/extensions_builder.go:59        Extension started.        {"kind": "extension", "name": "pprof"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/extensions_builder.go:53        Extension is starting...        {"kind": "extension", "name": "zpages"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        zpagesextension/zpagesextension.go:45        Host's zPages not available        {"kind": "extension", "name": "zpages"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        zpagesextension/zpagesextension.go:55        Starting zPages extension        {"kind": "extension", "name": "zpages", "config": {"TypeVal":"zpages",>
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/extensions_builder.go:59        Extension started.        {"kind": "extension", "name": "zpages"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        service/service.go:200        Starting exporters...
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/exporters_builder.go:92        Exporter is starting...        {"kind": "exporter", "name": "logging"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/exporters_builder.go:97        Exporter started.        {"kind": "exporter", "name": "logging"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        service/service.go:205        Starting processors...
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/pipelines_builder.go:51        Pipeline is starting...        {"pipeline_name": "metrics", "pipeline_datatype": "metrics"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/pipelines_builder.go:62        Pipeline is started.        {"pipeline_name": "metrics", "pipeline_datatype": "metrics"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/pipelines_builder.go:51        Pipeline is starting...        {"pipeline_name": "traces", "pipeline_datatype": "traces"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/pipelines_builder.go:62        Pipeline is started.        {"pipeline_name": "traces", "pipeline_datatype": "traces"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        service/service.go:210        Starting receivers...
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:70        Receiver is starting...        {"kind": "receiver", "name": "jaeger"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        static/strategy_store.go:201        No sampling strategies provided or URL is unavailable, using defaults        {"kind": "receiver", "name": "jaege>
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:75        Receiver started.        {"kind": "receiver", "name": "jaeger"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:70        Receiver is starting...        {"kind": "receiver", "name": "zipkin"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:75        Receiver started.        {"kind": "receiver", "name": "zipkin"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.767+0200        info        builder/receivers_builder.go:70        Receiver is starting...        {"kind": "receiver", "name": "prometheus"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        builder/receivers_builder.go:75        Receiver started.        {"kind": "receiver", "name": "prometheus"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        builder/receivers_builder.go:70        Receiver is starting...        {"kind": "receiver", "name": "otlp"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        otlpreceiver/otlp.go:94        Starting GRPC server on endpoint 0.0.0.0:4317        {"kind": "receiver", "name": "otlp"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        otlpreceiver/otlp.go:137        Setting up a second GRPC listener on legacy endpoint 0.0.0.0:55680        {"kind": "receiver", "name": "otlp"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        otlpreceiver/otlp.go:94        Starting GRPC server on endpoint 0.0.0.0:55680        {"kind": "receiver", "name": "otlp"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        otlpreceiver/otlp.go:112        Starting HTTP server on endpoint 0.0.0.0:55681        {"kind": "receiver", "name": "otlp"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        builder/receivers_builder.go:75        Receiver started.        {"kind": "receiver", "name": "otlp"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        builder/receivers_builder.go:70        Receiver is starting...        {"kind": "receiver", "name": "opencensus"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        builder/receivers_builder.go:75        Receiver started.        {"kind": "receiver", "name": "opencensus"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        healthcheck/handler.go:128        Health Check state change        {"kind": "extension", "name": "health_check", "status": "ready"}
Apr 28 14:51:41 guarana otelcol[262026]: 2021-04-28T14:51:41.950+0200        info        service/application.go:201        Everything is ready. Begin running and processing data.

```

Reply:
> feel free to pick whichever you'd prefer...

@slernersplunk Please make it consistent. I think `otelcol` fits better.

Reply:
done